src/serlo-editor/editor-ui/focus-helper.ts

```diff
@@ -0,0 +1,34 @@
+import { startsWith } from 'ramda'
+
+import { FocusPath } from '../types'
+
+interface PathAndFocus {
+  pluginId: string
+  path: Array<string | number>
+  targetPath?: Array<string | number>
+  currentFocus: FocusPath
```

Comment:
This interface reminds me of ZZ Top. In ZZ Top, there are two guys with huge beards, and one other guy. The other guy's last name is Beard.

Same here, we have `path` and `targetPath`, but it's the `currentFocus` whose type is `FocusPath`. 😁